stripe/checkoutCompleted.js

```diff
@@ -0,0 +1,59 @@
+import Stripe from 'stripe';
+import { v1 as uuidv1 } from 'uuid';
+import { badRequest, failure, success } from '../libs/response-lib';
+import * as dynamoDbLib from '../libs/dynamodb-lib';
+
+export const main = async event => {
+  const stripeSignature = event.headers['Stripe-Signature'];
+  const stripe = Stripe(process.env.STRIPE_API_SECRET_KEY);
+
+  let webhookEvent;
+
+  console.log('Verify the event with Stripe');
+  try {
+    webhookEvent = stripe.webhooks.constructEvent(
+      event.body,
+      stripeSignature,
+      process.env.STRIPE_WEBHOOK_CHECKOUT_COMPLETED_SECRET_KEY // We are getting this one from Stripe Dashboard, when a webhook is created
+    );
+  } catch (err) {
+    console.error(err);
+    return badRequest(`Webhook Error: ${err.message}`);
```

Comment:
I would expect `bad request` to mean `the parameters/header/body of the request you provided are incorrect`. here it could be some error inside the `constructEvent` method. ~Internal Server Error` seems more suitable in this ccase

Reply:
This is correct. Something is not right with the parameters/header/body from Stripe. That's why it's a bad request.